site/cypress/e2e/polls.spec.js

```diff
@@ -188,15 +188,17 @@ describe('Test cases revolving around polls functionality', () => {
         cy.get('h2').contains('Possible responses:');
         cy.get('.markdown').should('contain', 'Question goes here...?');
         cy.get('.markdown').should('not.contain', '#');
-        cy.get('.radio').eq(0).contains('No response');
-        cy.get('.radio > input').eq(0).should('be.disabled');
-        cy.get('.radio > input').eq(0).should('be.checked');
-        cy.get('.radio').eq(1).contains('Answer 1');
-        cy.get('.radio > input').eq(1).should('be.disabled');
-        cy.get('.radio').eq(2).contains('Answer 2');
-        cy.get('.radio > input').eq(2).should('be.disabled');
-        cy.get('.radio').eq(3).contains('Answer 3');
-        cy.get('.radio > input').eq(3).should('be.disabled');
+        // go through options, verify text and status of buttons
+        cy.get('table').contains('td','No response');
+        cy.get('table > tbody > tr:nth-child(1) > td:nth-child(2)').contains('No response');
+        cy.get('table > tbody > tr:nth-child(1) > td:nth-child(1) > input').should('be.disabled');
+        cy.get('table > tbody > tr:nth-child(1) > td:nth-child(1) > input').should('be.checked');
+        cy.get('table > tbody > tr:nth-child(2) > td:nth-child(2)').contains('Answer 1');
+        cy.get('table > tbody > tr:nth-child(2) > td:nth-child(1) > input').should('be.disabled');
+        cy.get('table > tbody > tr:nth-child(3) > td:nth-child(2)').contains('Answer 2');
+        cy.get('table > tbody > tr:nth-child(3) > td:nth-child(1) > input').should('be.disabled');
+        cy.get('table > tbody > tr:nth-child(4) > td:nth-child(2)').contains('Answer 3');
+        cy.get('table > tbody > tr:nth-child(4) > td:nth-child(1) > input').should('be.disabled');
```

Comment:
It might be better to target the table by its class instead.  As it is now, adding another table to this page could potentially break this test.

Reply:
Apply throughout.

Reply:
Sure. I think I've already set a class for the table, I think I can swap it on really quick.

Reply:
Was this test missed while making other changes?  The other tests target by `.poll-content`.

Reply:
They were missed. Oops.